src/main/java/mate/academy/mainspringproject/service/BookServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.mainspringproject.service;
+
+import java.util.List;
+import mate.academy.mainspringproject.model.Book;
+import mate.academy.mainspringproject.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+
+    private final BookRepository bookRepository;
+
+    @Autowired
+    public BookServiceImpl(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
```

Comment:
if we are using Lombok - lets use it
```suggestion
@Service
@RequiredArgsConstructor
public class BookServiceImpl implements BookService {

    private final BookRepository bookRepository;
```